src/utils/getTimeDifference.ts

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 
-export const getTimeDifference = (createdAt: Date): string => {
+export const getTimeDifference = (createdAt: string): string => {
   const now = new Date()
   const diffInSeconds = Math.floor((now.getTime() - new Date(createdAt).getTime()) / 1000)
```

Comment:
I remember seeing `date-fns` as a dependency long ago in package.json. If so, it would be easier to implement this using `formatDistance` (https://date-fns.org/v2.16.1/docs/formatDistance)